hack/golangci.yaml

```diff
@@ -118,7 +118,7 @@ linters:
 
       - linters:
           - govet
-        text: "lostcancel|printf"
+        text: "lostcancel"
```

Comment:
When enabling the `go vet printf` for `tCtx.Logf` and friends I was surprised to get a compile error but no output from `hack/verify-golangci-lint.sh` - this was why.